dashboard/src/actions/apps.ts

```diff
@@ -99,11 +104,10 @@ export function getApp(
   installedPackageRef?: InstalledPackageReference,
 ): ThunkAction<Promise<void>, IStoreState, null, AppsAction> {
   return async dispatch => {
-    dispatch(requestApps());
+    dispatch(requestApp());
```

Comment:
Are we going to unify terminology regarding "app" vs "package", etc? I mean, for example this function is called `getApp`, actions are `requestApp` and `selectApp` but it is dealing with `*PackageDetail` and outputting  `installedPackageDetail` and `availablePackageDetail`. Or am I not getting the distinction here (which could perfectly be 😅 ) ?

Reply:
I guess this is for historical reasons. I did perform a rename from `chart` to `package` back in time; however, the `apps` thing is something still pending.
+1 to move from `App` to `InstalledPackage` instead, of course.

Reply:
Yes, exactly. I did actually have it with package initially, but then saw the other actions related to `App` and `Apps`, and didn't want to create an inconsistency here nor rename all of them here. Happy to do a separate PR with such.